lib/types/models.ts

```diff
@@ -744,3 +744,23 @@ export interface SubscriptionAddonDiscount {
 	discount_percentage: number;
 	discounts__plan_addon: NavigationResource<PlanAddon>;
 }
+
+export interface DeviceHistory {
```

Comment:
You should also add a new row in `ResourceTypeMap` like:
```
device_history: DeviceHistory;
```
This is the resource-to-interface map that the SDK typings use to enable fully typed results.